File: scala/app/controllers/Pilcrows.scala
package controllers

import play.api.mvc.Controller
import play.api.mvc.Action

object Pilcrows extends Controller {
  
  def index = Action {
    Ok(views.html.index())
  }
  
  def create = Action {
	Ok(views.html.create())
  }
  
  def search = Action {
	Ok(views.html.index())
  }
  
  def edit = Action {
	Ok(views.html.edit())
  }  
}

File: java/app/controllers/Pilcrows.java
package controllers;

import models.Quote;
import play.mvc.Controller;
import play.mvc.Result;

import java.util.ArrayList;
import java.util.List;

public class Pilcrows extends Controller {

    public static Result index() {
        return ok(views.html.index.render(Quote.find.all()));
    }

    public static Result create() {
        return ok(views.html.create.render());
    }

    public static Result search() {
        return ok(views.html.index.render(new ArrayList<Quote>()));
    }
    
    public static Result edit() {
        return ok(views.html.edit.render());
    }
}
